src/main/java/frc/robot/components/TrigonXboxController.java

```diff
@@ -21,86 +21,68 @@
 
     public TrigonXboxController(int port) {
         super(port);
-        buttonA = new JoystickButton(this, Button.kA.value);
-        buttonB = new JoystickButton(this, Button.kB.value);
-        buttonX = new JoystickButton(this, Button.kX.value);
-        buttonY = new JoystickButton(this, Button.kY.value);
-        leftBumper = new JoystickButton(this, Button.kBumperLeft.value);
-        rightBumper = new JoystickButton(this, Button.kBumperRight.value);
-        leftStickButton = new JoystickButton(this, Button.kStickLeft.value);
-        rightStickButton = new JoystickButton(this, Button.kStickRight.value);
+        a = new JoystickButton(this, Button.kA.value);
+        b = new JoystickButton(this, Button.kB.value);
+        x = new JoystickButton(this, Button.kX.value);
+        y = new JoystickButton(this, Button.kY.value);
+        leftBumper = new JoystickButton(this, Button.kLeftBumper.value);
+        rightBumper = new JoystickButton(this, Button.kRightBumper.value);
+        leftStickButton = new JoystickButton(this, Button.kLeftStick.value);
+        rightStickButton = new JoystickButton(this, Button.kRightStick.value);
 
         backButton = new JoystickButton(this, Button.kBack.value);
         startButton = new JoystickButton(this, Button.kStart.value);
         rumbleAmount = -1;
         notifier = new Notifier(this::notifierPeriodic);
     }
 
-    public JoystickButton getButtonA() {
-        return buttonA;
+    public JoystickButton getABtn() {
```

Comment:
Btn is not short for button

Reply:
trust me, it is

Reply:
no

Reply:
![image](https://user-images.githubusercontent.com/29277997/152160712-400ab5db-f03b-40ec-8744-32b15b1af879.png)

Reply:
but it's disgusting

Reply:
btn is not